2014-09-04 11:50:26  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-1:1407109 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-04 15:20:34  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-4:608631 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-04 15:20:41  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-5:615857 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-04 15:21:16  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-1:650129 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-04 15:22:09  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-2:40064 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-04 15:22:26  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-3:57192 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-04 15:22:26  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-4:57193 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-04 15:22:28  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-5:58618 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-04 15:22:29  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-1:59708 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-04 15:25:31  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-4:242369 ] - [ ERROR ]  the handler is exception +org.apache.mina.core.buffer.BufferDataException: java.io.NotSerializableException: org.apache.mina.transport.socket.nio.NioSocketSession
2014-09-04 15:26:38  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-5:308646 ] - [ ERROR ]  the handler is exception +org.apache.mina.core.buffer.BufferDataException: java.io.NotSerializableException: org.apache.mina.transport.socket.nio.NioSocketSession
2014-09-04 15:26:57  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-1:327817 ] - [ ERROR ]  the handler is exception +org.apache.mina.core.buffer.BufferDataException: java.io.NotSerializableException: org.apache.mina.transport.socket.nio.NioSocketSession
2014-09-04 15:28:01  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-3:391606 ] - [ ERROR ]  the handler is exception +org.apache.mina.core.buffer.BufferDataException: java.io.NotSerializableException: org.apache.mina.transport.socket.nio.NioSocketSession
2014-09-04 15:28:09  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-4:400390 ] - [ ERROR ]  the handler is exception +org.apache.mina.core.buffer.BufferDataException: java.io.NotSerializableException: org.apache.mina.transport.socket.nio.NioSocketSession
2014-09-04 15:42:45  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-2:7174 ] - [ ERROR ]  the handler is exception +org.apache.mina.core.buffer.BufferDataException: java.io.NotSerializableException: org.apache.mina.transport.socket.nio.NioSocketSession
2014-09-04 15:43:05  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-4:27277 ] - [ ERROR ]  the handler is exception +org.apache.mina.core.buffer.BufferDataException: java.io.NotSerializableException: org.apache.mina.transport.socket.nio.NioSocketSession
2014-09-04 15:44:41  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-5:123705 ] - [ ERROR ]  the handler is exception +org.apache.mina.core.buffer.BufferDataException: java.io.NotSerializableException: org.apache.mina.transport.socket.nio.NioSocketSession
2014-09-04 15:46:05  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-1:207293 ] - [ ERROR ]  the handler is exception +org.apache.mina.core.buffer.BufferDataException: java.io.NotSerializableException: org.apache.mina.transport.socket.nio.NioSocketSession
2014-09-04 15:46:14  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-2:215966 ] - [ ERROR ]  the handler is exception +org.apache.mina.core.buffer.BufferDataException: java.io.NotSerializableException: org.apache.mina.transport.socket.nio.NioSocketSession
2014-09-04 15:47:46  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-4:308565 ] - [ ERROR ]  the handler is exception +org.apache.mina.core.buffer.BufferDataException: java.io.NotSerializableException: org.apache.mina.transport.socket.nio.NioSocketSession
2014-09-04 15:47:47  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-3:309686 ] - [ ERROR ]  the handler is exception +org.apache.mina.core.buffer.BufferDataException: java.io.NotSerializableException: org.apache.mina.transport.socket.nio.NioSocketSession
2014-09-04 15:50:07  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-2:449077 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-04 15:58:27  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-3:948924 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-04 15:58:28  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-4:949886 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-04 15:58:32  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-5:954270 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-04 15:58:38  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-1:960144 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-04 15:58:54  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-5:975813 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-04 15:58:58  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-1:980458 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-04 15:59:04  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-3:986101 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-04 16:02:31  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-3:1192949 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-04 16:02:31  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-4:1192951 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-04 16:03:16  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-3:1238722 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-04 16:35:39  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-4:3181038 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
2014-09-04 16:35:48  [ org.huangxu.service.ServerHandler.exceptionCaught(ServerHandler.java:37):org.huangxu.service.ServerHandler:NioProcessor-1:3189807 ] - [ ERROR ]  the handler is exception +An existing connection was forcibly closed by the remote host
